src/main/java/com/softserve/academy/spaced/repetition/controller/RegistrationController.java

```diff
@@ -1,7 +1,7 @@
 package com.softserve.academy.spaced.repetition.controller;
 
 
-import com.softserve.academy.spaced.repetition.controller.dto.impl.NewAccountPasswordDTO;
+import com.softserve.academy.spaced.repetition.controller.dto.simpleDTO.NewAccountPasswordDTO;
```

Comment:
Why did you rename or move dtos?

Reply:
because it`s not a impl of DTO

Reply:
Okay, i agree, but was it a part of your task?